tests/src/Common/CoreFX/CoreFX.depproj

```diff
@@ -13,7 +13,8 @@
       <SystemCompositionVersions>1.3.0-preview3-26501-04</SystemCompositionVersions>
       <XUnitNetcoreExtensionsVersion>2.2.0-preview1-02902-01</XUnitNetcoreExtensionsVersion>
       <!-- Xunit Abstractions is differently versioned then the rest of XUnit -->
-      <XUnitAbstractionsVersion>2.0.1</XUnitAbstractionsVersion>
+      <XUnitAbstractionsVersion>2.0.3</XUnitAbstractionsVersion>
+      <XUnitAssertVersion>2.4.1-pre.build.4059</XUnitAssertVersion>
```

Comment:
Thanks, Jan.  I assume there was a change in Assert.Equal where previously (in the version coreclr has) it was assuming GetType() == GetType(), and now it's not requiring that?

Reply:
Right, here is the related change from xunit: https://github.com/xunit/assert.xunit/commit/a42b055ae6fbaf2b54df079bbf3c51404a99d2c7#diff-445c3deeda8978a1cfcb9ec2b9f45d8eL52